apps/glusterfs/app_volume.go

```diff
@@ -95,6 +95,18 @@ func (a *App) VolumeCreate(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	for _, volumeOption := range msg.Options {
+		switch volumeOption {
+		case "encryption":
```

Comment:
@humblec and  I have addressed this comment. check for the heketi-cli volume create -h. 
We can have our own list of strings(like "encryption") which chooses options related to string or they can pass the option directly. volume create is failed if they mentioned a wrong volume option. 
Added functional test.